File: 백준/Bronze/2441. 별 찍기 － 4/별 찍기 － 4.java
import javax.script.ScriptContext;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int count = sc.nextInt();

        for(int i = count ; i> 0; i--){
            for(int j = count ;j > i; j--){
                System.out.printf(" ");
            }
            for(int j = 0; j < i ; j++){
                System.out.printf("*");
            }
            System.out.println();
        }
    }
}

File: 백준/Bronze/2441. 별 찍기 － 4/별 찍기 － 4.cc
#include <iostream>
using namespace std;

int main() {
    int count;
    cin >> count;

    for(int i = count; i >0; i--){
        for(int j = count-i; j >0; j--){
            cout << " ";
        }
        for(int j = i; j >0; j--){
            cout << "*";
        }
        cout << endl;
    }
}